ui/nuxeo-document-comments/nuxeo-document-comments-mixin.js

```diff
@@ -0,0 +1,70 @@
+import { html } from '@polymer/polymer/lib/utils/html-tag.js';
+
+const CommentsMixin = (superClass) =>
+  class extends superClass {
+    static get properties() {
+      return {
+        text: {
+          type: String,
+          value: '',
+        },
+      };
+    }
+
+    static get inputTemplate() {
+      return html`
+        <div class="input-area">
+          <paper-textarea
+            id="inputContainer"
+            placeholder="[[_computeTextLabel(level, 'writePlaceholder', null, i18n)]]"
+            value="{{text}}"
+            max-rows="[[_computeMaxRows()]]"
+            no-label-float
+            on-keydown="_checkForEnter"
+          >
+          </paper-textarea>
+          <template is="dom-if" if="[[!_isBlank(text)]]">
+            <iron-icon
+              id="submit"
+              name="submit"
+              class="main-option opaque"
+              icon="check"
+              on-tap="_submitComment"
```

Comment:
I'd rather define an "abstract" `_submitComment` here (with a comment)  for reference